docs/modules/knowledge-graph/pages/knowledge-store.adoc

```diff
@@ -0,0 +1,47 @@
+= {graph-store}
+
+{graph-store} is a hybrid vector-and-graph store that combines the benefits of vector stores with the context and relationships of related edges between chunks.
+
+See the xref:examples:knowledge-store.adoc[graph store example code] to get started with {graph-store}.
+
+[IMPORTANT]
+====
+This feature is currently under development and has not been fully tested. It is not supported for use in production environments. Please use this feature in testing and development environments only.
+====
+
+== The `ragstack-ai-knowledge-store` library
+
+The `ragstack-ai-knowledge-store` library contains functions for creating a hybrid vector-and-graph knowledge store. This store combines the benefits of vector stores with the context and relationships of a related edges.
+
+To install the package, run:
+
+[source,bash]
+----
+pip install ragstack-ai-knowledge-store
+----
+
+== What's the difference between fine-grained and coarse-grained knowledge graphs?
```

Comment:
May want to update terms to reflect the "entity-centric" and "content-centric" terminology we've used in the blog post. Fine-grained vs. Coarse-grained may be a bit too vague?